Lib/ipaddress.py

```diff
@@ -2377,6 +2377,7 @@ class _IPv6Constants:
         IPv6Network('2002::/16'),
         IPv6Network('fc00::/7'),
         IPv6Network('fe80::/10'),
+        IPv6Network('3fff::/20'),
```

Comment:
Can you please add a comment with a reference to RFC 9637?

Can you move it between 2002 and fc00?